README.md

```diff
@@ -168,7 +168,7 @@ shoryuken [options]
     -c, --concurrency INT            Processor threads to use
     -d, --daemon                     Daemonize process
     -q, --queue QUEUE[,WEIGHT]...    Queues to process with optional weights
-    -r, --require [PATH|DIR]         Location of the worker
+    -r, --require [PATH]             Location of the worker
```

Comment:
@phstc yeah I agree it would be good to check if it is a dir and load all files inside. Will create a PR for that.